pkg/storage/driver/secrets.go

```diff
@@ -98,7 +99,7 @@ func (secrets *Secrets) List(filter func(*rspb.Release) bool) ([]*rspb.Release,
 			continue
 		}
 
-		rls.Labels = item.ObjectMeta.Labels
+		rls.Labels = filterSystemLabels(item.ObjectMeta.Labels)
```

Comment:
This change appears to have caused a breakage of the `helm list --selector` feature, see: https://github.com/helm/helm/issues/12435

Reply:
So desired behaviour is allowing users to filter using system labels?

Reply:
Not a maintainer, but I imagine this can indeed by useful to select a subset of things using the default labels Helm provides you with.

It also appears to be what's being tested, which continued to work because the in-memory driver was not modified in this PR.

Reply:
Let's continue this discussion in fix PR - https://github.com/helm/helm/pull/12447